packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Paper, Typography } from '@mui/material';
+
+import { LogoLarge, LogoText } from '../Branding/branding';
+import { Button } from '../Button';
+import { ReactComponent as SadFace } from './sad-face.svg';
+
+const styles = {
+  paper: {
+    display: 'flex',
+    flexDirection: 'column',
+    p: 6,
+    ['@media (max-height:600px)']: {
+      p: 4,
+    },
+  },
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+  },
+  sadFace: {
+    width: 250,
+    height: 250,
+    alignSelf: 'center',
+    ['@media (max-height:600px)']: {
+      width: 150,
+      height: 150,
+    },
+  },
+  typography: {
+    margin: '10px',
+  },
+  logo: {
+    height: 74,
+    width: 76,
+    ['@media (max-height:600px)']: {
+      height: 53,
+      width: 56,
+    },
+  },
+  logoText: {
+    width: 116,
+    height: 34,
+  },
+};
+
+export interface Props {
+  onNavigateHome: (e: React.BaseSyntheticEvent) => void;
+  fitContainer: boolean;
+}
+
+const PageNotFound = ({ onNavigateHome, fitContainer }: Props) => (
+  <Paper
+    sx={{
+      minWidth: `100${fitContainer ? '%' : 'vw'}`,
+      minHeight: `100${fitContainer ? '%' : 'vh'}`,
+      ...styles.paper,
+    }}
+    elevation={0}
+    square={true}
+  >
+    <Box sx={{ flex: '1 1 0', ...styles.container }}>
+      <Box>
+        <LogoLarge sx={styles.logo} />
+      </Box>
+      <SadFace sx={styles.sadFace} />
+      <Typography variant="h3" align="center" sx={styles.typography}>
+        Page not found!
```

Comment:
This text, and the paragraph beneath it, should not be capitalized according to the Figma design.  Might be able to add `text-transform: none` to the styles to fix this.

Reply:
Good find. I set `text-transform: none` onto `styles.typography`